src/app/faqs/faq-list/faq-list.component.html

```diff
@@ -1,142 +1,139 @@
-<div class="d-flex row-hl">
-  <!-- Category Selector -->
-  <div class="p-2 w-20">
-    <mat-card
-      class="mb-3 clickable"
-      matRipple
-      (click)="onClickTopCategory()"
-      [ngClass]="topOnly ? 'background-primary' : ''"
-    >
-      <mat-card-title>
-        <i class="fa fa-star"></i>
-        Top Questions</mat-card-title
-      >
-      <mat-card-subtitle
-        >The top questions we receive across all categories</mat-card-subtitle
-      >
-      <mat-card-content></mat-card-content>
-    </mat-card>
-    <hr />
-    <div *ngFor="let faqCategory of faqCategories">
+<div fxLayout="row" fxLayoutAlign="center center" class="k-header">
+  <button
+    mat-fab
+    color="primary"
+    matTooltip="Add Category"
+    (click)="onClickAddCategory()"
+  >
+    <mat-icon>add</mat-icon>
```

Comment:
My instinct for this FAB would be that it creates a FAQ, not a Category. Might confuse users.

Reply:
Agree. After some thought I'd like to move the 'add category' into the form itself. Either pick from an existing category or add a new one. I've added it to KP-30